packages/react-kitchen-sink/src/react-query/FirestoreSnaphotManager.ts

```
@@ -1,15 +1,45 @@
+  type DocumentData,
+  type DocumentReference,
+  type Query as FirestoreQuery,
+  type SnapshotListenOptions,
+} from '@firebase/firestore'
+import {
+  hashKey,
+  type Query as TanstackQuery,
+  type QueryCacheNotifyEvent,
+  type QueryClient,
+  type QueryKey,
+} from '@tanstack/react-query'
```

ESLint

[error] Unable to resolve path to module '@tanstack/react-query'.

(import-x/no-unresolved)